orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -239,3 +241,47 @@ def test_model_to_device_and_back_to_original(original_device, to_device):
     assert model._device == torch.device(original_device+':0')
     for _, parameter_value in model.named_parameters():
         assert parameter_value.device.type == original_device
+
+def test_gpu_reserved_memory_with_torch_no_grad():
+    device = 'cuda'
+
+    config = AutoConfig.from_pretrained(
```

Comment:
This model is actually quiet useful. You could put it on a helper function for reuse